schemas/fingerprint-server-api.yaml

```diff
@@ -1227,6 +1226,7 @@ components:
                       example:
                         - Ibk1527CUFmcnjLwIs4A
                   required:
+                    - tag
```

Comment:
Schama lint is failing on https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/actions/runs/10180403441/job/28158190592?pr=115

> `required` field `tag` is not defined in `properties` 

The linter can't evaluate our `allOf` transformers, I suppose we need to duplicate the `tag` field here?

Reply:
Fixed